app/frontend/src/pages/Demo/Demo.tsx

```diff
@@ -14,9 +16,55 @@ function Demo() {
   function checkboxOnChange(e: React.ChangeEvent<HTMLInputElement>) {
     console.log(e.target.checked);
   }
-
+  console.log("changes");
   return (
     <Fragment>
+      <HeaderNav
+        logoDesktop={logoHorizontal}
+        logoMobile={logoStacked}
+        menu={[
+          { name: "Hack for LA", link: "/" },
+          { name: "How to Join", link: "/qualifier" },
+          { name: "Projects", link: "/demo" }
+        ]}
+      />
+      <NotificationBar
+        text={
```

Comment:
I'm surprised typescript did not catch this, since these are not strings, theoretically. But if we plan to insert something more complicated that string into a `text` prop, using children would be the way to go. Children generally gives more freedom for the developer to insert complex content into a component.